src/RestAPI/dlwsrestapi.py

```diff
@@ -597,12 +597,29 @@ def get(self):
 class AddUser(Resource):
     def get(self):
         parser.add_argument('userName')
-        parser.add_argument('userId')
+        parser.add_argument('uid')
+        parser.add_argument('gid')
+        parser.add_argument('groups')
         args = parser.parse_args()
-        username = args["userName"]
-        userId = args["userId"]
+
         ret = {}
-        ret["status"] = JobRestAPIUtils.AddUser(username,userId)
+        userName = args["userName"]
+        if args["uid"] is None or len(args["uid"].strip()) == 0:
+            uid = 0
```

Comment:
uid & gid cannot be 0. It becomes root...

Reply:
Fixed.